timelapse/__main__.py

```diff
@@ -11,55 +11,55 @@
 from Foundation import NSUserDefaults
 
 
-def dark_mode():
+def dark_mode() -> Any:
```

Comment:
shouldn't that be `bool`?

Reply:
ahhh I missed the `==`. I wasn't sure what type it was. Thanks! Fixed in https://github.com/mre/timelapse/pull/33/commits/040a50f53d9034923eb94ee0caa24bd25028eebc